selfdrive/test/process_replay/ref_commit

```diff
@@ -1 +1 @@
-158743f002e82b62a67c6b91308db01d715f1643
\ No newline at end of file
+aed38917ed4c187c6f8d9607c245d42d614371e1
```

Comment:
why did this change?

Reply:
Thought it was a standard procedure for ports now. Will revert.

Reply:
I see now that it's related to logging/cereal. Thanks for pointing this out.